google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1498,6 +1503,43 @@ def compose sources, destination, acl: nil, encryption_key: nil
         #                                  issuer: "service-account@gcloud.com",
         #                                  signing_key: key
         #
+        # @example Using the `issuer` and `signer` options:
```

Comment:
What about mentioning the IAM request in the title? For example: `Using Cloud IAM `signBlob` to create the signature:`